napari/_vispy/experimental/_tests/test_vispy_tiled_image.py

```diff
@@ -221,3 +221,48 @@ def test_tiled_single_scale(qtbot, monkeypatch, make_napari_viewer):
     np.testing.assert_allclose(
         screenshot[-screen_offset, -screen_offset], target_center
     )
+
+
+@pytest.mark.async_only
+@pytest.mark.skip("NAPARI_OCTREE env var cannot be dynamically set")
```

Comment:
@sofroniewn what does this mean? ie what's the point of everything else :joy:

Reply:
ie: why not skip *only* if NAPARI_OCTREE is not externally set?

Reply:
It's really just a place holder for now until we figure out how to run these on CI, but I should make a PR that allows them to be run locally if the NAPARI_OCTREE is externally set (same as what you said). I'll do that in a followup for all tests in this file